BassBotDotNET/Program.cs

```diff
@@ -4,14 +4,23 @@
 using Discord.WebSocket;
 using Newtonsoft.Json;
 
+namespace BassBotDotNET;
+
+/// <summary>
+/// Main program to run the discord bot.
+/// </summary>
```

Comment:
Oh do you comment by using 3 positive slashes? And some of the comments remind me of html.

Reply:
two slashes can make a comment. but the formatting for method/class documentation looks like this ^